### PR TITLE
Bump container-definitions dependency to prerelease 0.49.1000-69584

### PR DESCRIPTION
**Pull request by markfields**
Bump container-definitions dependency to prerelease 0.49.1000-69584

## Description

This brings https://github.com/microsoft/FluidFramework/pull/10416 to the client, and updates client packages accordingly.  No functional change.

## PR Checklist

> Use the check-list below to ensure your branch is ready for PR. If the item is not applicable, leave it blank.

-   [x] I have updated the documentation accordingly. (need to update client BREAKING.md)
-   [x] ~I have added tests to cover my changes.~
-   [ ] All new and existing tests passed.
-   [x] My code follows the code style of this project.
-   [x] I ran the lint checks which produced no new errors nor warnings for my changes.
-   [x] I have checked to ensure there aren't other open Pull Requests for the same update/change.

## Does this introduce a breaking change?

-   [X] Yes
-   [ ] No

> If this introduces a breaking change, please describe the impact and migration path for existing applications below.

## Testing

N/A

## Any relevant logs or outputs

N/A

## Other information or known dependencies

Need to open some follow-up tasks in our backlog to track starting to use new `EstablishingConnection` state.

### CODE REVIEW

File: packages/loader/container-loader/src/container.ts [hunk 1]
```diff
@@ -147,19 +147,33 @@ export interface IContainerConfig {
...
+     * The container is disconnected but actively trying to establish a new connection
+     * PLEASE NOTE that this numerical value falls out of the order you may expect for this state
      */
-    Connecting,
+    EstablishingConnection = 3,
 
     /**
-     * The document is fully connected
+     * @see ConnectionState.CatchingUp, which is the new name for this state.
+     * @deprecated - This state itself is not gone, just being renamed. Please use ConnectionState.CatchingUp.
```
Comment (scarlettjlee) [inline packages/loader/container-loader/src/container.ts:164]: Does this need a breaking.md note?

Comment (markfields) [reply packages/loader/container-loader/src/container.ts:164]: Yeah, coming soon™️  Thanks for taking a peek!

**Comment by markfields** [comment]
@tylerbutler If I'm not allowed to merge this into `next`, then I think this is blocked until you sort out the issues we discussed earlier you faced when updating client packages in `main` to depend on pre-release `container-definitions` 0.49.1000.

It will make the release go smoother if I merge this before you start the release though, so keep in touch.

**Comment by tylerbutler** [comment]
@markfields Can you try retargeting to main and see what happens in CI?

File: packages/loader/container-loader/src/container.ts (hunk 1 above)
Comment (markfields) [reply packages/loader/container-loader/src/container.ts]: See #10535

**Comment by markfields** [comment]
For posterity. see https://github.com/microsoft/FluidFramework/pull/10447/files/db270d943c73aa894e0383224c2c97d6680ea93b for the original state of this PR.

It was properly included in #10468 (see latest commit and how there are basically no changes to prove it - that's after merging in `next`, there were no interesting merge conflicts).  It was missing BREAKING.md notes, which are added in https://github.com/microsoft/FluidFramework/pull/10535

**Pull request closed without merging**